plasmapy/particles/particle_class.py

```diff
@@ -111,13 +145,44 @@ def mass(self) -> Union[u.Quantity, Real]:
     def charge(self) -> Union[u.Quantity, Real]:
         raise NotImplementedError
 
+    @property
+    def json_dict(self) -> dict:
+        """
+        Provides the standard format for JSON representation of particles which
+        inherit from AbstractParticle.
+
+        Keys:
+            plasmapy_particle - Specifies that this JSON is for some particle
+                type - The type of the particle (e.g Particle or CustomParticle)
+                module - Specifies the module where the class was declared
+                date_created - Presents the UTC date and time of serialization
+                __init__ - contains a dictionary of the arguments used to create the particle
+                    args - non keyword arguments used to create a particle
+                    kwargs - keyword arguments used to create a particle
+        """
+        json_dictionary = {
+            "plasmapy_particle": {
+                "type": type(self).__name__,
+                "module": self.__module__,
+                "date_created": datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S UTC"),
+                "__init__": {"args": (), "kwargs": {},},
+            }
+        }
+        return json_dictionary
+
     def __bool__(self):
         """
         Raise an `~plasmapy.utils.AtomicError` because particles
         do not have a truth value.
         """
         raise AtomicError("The truth value of a particle is not defined.")
 
+    def to_json(self) -> str:
```

Comment:
```suggestion
    def json_dumps(self) -> str:
```

As I've mentioned in previous comments, a `json.dumps` method should be in addition to a `json.dump` method.  These method should also be explicitly named to reflect dumps vs dump.